uqcsbot/scripts/xkcd.py

```diff
@@ -89,7 +89,7 @@ def handle_xkcd(command: Command) -> None:
     bot.post_message(command.channel_id, response, unfurl_links=True, unfurl_media=True)
 
 
-@bot.on_schedule('cron', hour=14, minute=1, day_of_week='mon,wed,fri',
+@bot.on_schedule('cron', minute=5, day_of_week='mon,wed,fri',
```

Comment:
This will run the function whenever the minutes is 5 i.e. five past every hour. Probably not what you want.